Makefile

```diff
@@ -3,7 +3,7 @@ BINARY_NAME=gradetool
 REGISTRY?=quay.io
 TNF_IMAGE_NAME?=testnetworkfunction/gradetool
 IMAGE_TAG?=latest
-GOLANGCI_VERSION=v1.55.1
+GOLANGCI_VERSION=v1.55.2
```

Comment:
This should be a separate PR correct?

Reply:
The linter fails when not updated [0].

[0] https://github.com/golangci/golangci-lint/issues/3718